### ISSUE TITLE
TensorRT8 Pad op assert "[layers.cpp::setMode::3848] Error Code 3: Internal Error (Parameter check failed at: optimizer/api/layers.cpp::setMode::3848, condition: int(mode) >= 0 && int(mode) < EnumMax<SliceMode>()"

### ISSUE THREAD

**Issue by jianxiezhendong** [NVIDIA/TensorRT#1643]
## Description

I have a onnx model having a padding operation. When onnxparser runs, an error has occurred from the function("layer->setMode(nvinfer1::SliceMode::kFILL)") . The error is "[layers.cpp::setMode::3848] Error Code 3: Internal Error (Parameter check failed at: optimizer/api/layers.cpp::setMode::3848, condition: int(mode) >= 0 && int(mode) < EnumMax<SliceMode>()". 
If I should do some change generating the onnx file? Could you give me some advise?

## Environment

**TensorRT Version**:  8.0.3
**NVIDIA GPU**:  RTX 2080Ti
**NVIDIA Driver Version**:  470
**CUDA Version**:  11.4
**CUDNN Version**:  8.2
**Operating System**:  20.04
**Python Version (if applicable)**:  3.8
**Tensorflow Version (if applicable)**:  None
**PyTorch Version (if applicable)**:  None
**Baremetal or Container (if so, version)**:  Container

**Comment by jianxiezhendong** [comment]
I know why it occurs this error. The SliceMode is not match.

**Comment by truncs** [comment]
@jianxiezhendong how did you resolve this issue?

**Comment by AlexandreBourrieau** [comment]
same probleme how do you resolve this please ?
